eqdsk/cli/__init__.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-FileCopyrightText: 2023-present The Bluemira Developers <https://github.com/Fusion-Power-Plant-Framework/bluemira>
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+"""Eqdsk CLI"""
+
+import click
+import numpy as np
+
+from eqdsk.file import EQDSKInterface
+from eqdsk.log import eqdsk_banner, eqdsk_warn
+from tests._helpers import read_strict_geqdsk  # noqa: PLC2701
+
+
+def _setup_plotting(eq: EQDSKInterface):
+    try:
+        import matplotlib.pyplot as plt  # noqa: PLC0415
+    except ImportError:
+        raise ImportError(
+            "Matplotlib not installed, cannot plot.\n"
+            "To resolve, try running:\n"
+            "pip install matplotlib"
+        ) from None
+
+    fig, ax = plt.subplots()
+    fig.set_size_inches(8, 8)
+
+    ax.set_title(f"EQDSK:  {eq.name}")
+    ax.set_xlabel("R [m]")
+    ax.set_ylabel("Z [m]")
+
+    # plot mag center
+    ax.plot(eq.xmag, eq.zmag, color="red", marker="o")
+
+    return fig, ax, plt.show
+
+
+@click.group()
+@click.version_option()
+def cli():
+    """EQDSK cli
+
+    A useful utility for displaying eqdsk file data.
+    """
+
+
+@cli.command("show", no_args_is_help=True)
+@click.argument("filepath", type=click.Path(exists=True))
+def show(filepath):
+    """Reads and prints important parameters of the eqdsk."""
+    eq = EQDSKInterface.from_file(filepath, no_cocos=True)
+
+    try:
+        # tests if it is geqdsk readable/compatible
+        read_strict_geqdsk(filepath)
+        geqdsk_compatible = True
+    except Exception:  # noqa: BLE001
```

Comment:
Let get the actual exception(s)

Reply:
also this will never work on json files, so I feel like this is maybe something we don't need?